alidock/__init__.py

```diff
@@ -108,8 +116,9 @@ def run(self):
         userName = getpwuid(userId).pw_name
         initShPath = os.path.join(outDir, ".alidock-init.sh")
         with open(initShPath, "w") as fil:
-            fil.write(initSh.render(sharedDir=self.dirInside,
-                                    logRelative=self.logRelative,
+            fil.write(initSh.render(logRelative=self.logRelative,
+                                    sharedDir=self.dirInside,
+                                    dockName=self.conf["dockName"].rsplit("-", 1)[0],
```

Comment:
If you specify `--name <container>` you will get `[container]` in the shell prompt. You can effectively run/manage different containers at the same time (but beware, keep homes separated as well).